client/src/document/organisms/sidebar/index.tsx

```diff
@@ -7,7 +7,15 @@ import { useUIStatus } from "../../../ui-context";
 import "./index.scss";
 import { TOC } from "../toc";
 
-export function SidebarContainer({ doc, children }) {
+export function SidebarContainer({
+  doc,
+  label = undefined,
```

Comment:
tiny nit: is the `= undefined` required here?

Reply:
We also use the `SidebarContainer` and would then need to provide a `label` there as well. If I make it `= ""`, then React would add `aria-label=""`. `undefined` causes React to omit the attribute.

Reply:
Oh, no, you're probably right. I had added the typing afterwards.

Reply:
Fixed in aea17e23b.